cypress/page/products-list.page.ts

```diff
@@ -3,16 +3,16 @@ class ProductsListPage {
   private checkout: string;
 
   constructor (){
-    this.addToCartButton = "#center_column a.button.ajax_add_to_cart_button.btn.btn-default";
-    this.checkout = "[style*='display: block;'] .button-container > a";
+    this.addToCartButton = "a[title='Add to cart'";
```

Comment:
I think a `]` is missing for the addToCartButton

Reply:
yep